externs/shaka/manifest.js

```diff
@@ -128,6 +129,9 @@ shaka.extern.InitDataOverride;
  *   <i>Defaults to false.  Can be filled in by advanced DRM config.</i> <br>
  *   True if the application requires the key system to support persistent
  *   state, e.g., for persistent license storage.
+ * @property {Array.<string>} sessionTypes
```

Comment:
I wanted to keep the same field names as the MediaKeySystemAccess configuration, to keep consistence between the DRMEngine configuration, the given configuration to navigator APIs and the one we receive from them. 

Is that okay for you ?

Reply:
The entire array is passed to requestMediaKeySystemAccess, but only using sessionTypes[0] still seems like an issue to me.  EME says that all session types in the array are considered required in the query.  But what is the purpose of the second and subsequent members of the array if we ignore them?

Having an array implies that we will use all the values _somehow_.  I don't want to field bug reports later that say things like "I asked for temporary & persistent, but only got temporary sessions!  I wanted you to read my mind and do temporary for _these_ keys and persistent for _those_ keys."  And if we later decide that this was a mistake, and we should represent the config as a single value (since that's all we can use), we would have to go to Shaka v4 before we could drop the more confusing version of the config.

The API shape of Shaka Player (including the config) should reflect what we will _actually use_, rather than exactly how we will pass the values to an underlying browser API.

Reply:
Your point makes total sense, I'll change it